proto/src/main/proto/Rss.proto

```diff
@@ -251,7 +251,6 @@ message ShuffleServerHeartBeatRequest {
   int64 availableMemory = 4;
   int32 eventNumInFlush = 5;
   repeated string tags = 6;
-  google.protobuf.BoolValue isHealthy = 7;
```

Comment:
Don't modify the field. You can add a comment `// deprecated`. If you modify this,  this will be incompatible feature. Although we don't release 1.0 version, we don't need guarantee the compatibility, I still hope we reduce the similar breaking change.

Reply:
Okay, I'll recover